librz/core/canalysis.c

```diff
@@ -7262,6 +7262,22 @@ RZ_IPI char *rz_core_analysis_all_vars_display(RzCore *core, RzAnalysisFunction
 	return rz_strbuf_drain(sb);
 }
 
+RZ_API bool rz_analysis_add_io_registers_map(RzIO *io, RzAnalysis *analysis, ut64 file_offset) {
+	ut64 rom_size = analysis->arch_target->profile->rom_size;
+	ut64 rom_address = analysis->arch_target->profile->rom_address;
+	RzIOMap *map = rz_io_map_get(io, rom_address);
+	if (!map) {
+		return false;
+	}
+	if (file_offset) {
+		eprintf("Warning: Not mapping the ROM section due to the mapping at 0x%llx \n", file_offset);
```

Comment:
`RZ_LOG_WARN("Cannot map the ROM section at %" PFMT64x " with a size of %" PFMT64x " due file being mapped at 0x%" PFMT64x "\n", rom_address, rom_size, file_offset);`